File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F11.P1.go
package main

import "fmt"

func readAt(xs []int, i int) int {
	// BEGIN PROBE F11.P1
	e := xs[i]
	return e
	// END PROBE F11.P1
}

func main() {
	fmt.Println(readAt([]int{1, 2, 3}, 2))
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/rust/F11.P1.rs
fn read_at(xs: Vec<i32>, i: i32) -> i32 {
// BEGIN PROBE F11.P1
let e = xs[i as usize];
e
// END PROBE F11.P1
}

fn main() {
    println!("{}", read_at(vec![1, 2, 3], 2));
}
